modules/mixcr/1.0/config/default.yaml

```diff
@@ -0,0 +1,37 @@
+lcr-modules:
+    
+    mixcr:
+
+        inputs:
+            # Available wildcards: {seq_type} {genome_build} {sample_id}
+            sample_fastq_1: "__UPDATE__" 
+            sample_fastq_2: "__UPDATE__"
+
+        scratch_subdirectories: []
+
+        options:
+            mixcr_run: 
+                mrna:
+                    --starting-material rna
+                genome:
+                    --starting-material dna
+
+        # TODO: Add the conda environment once MiXCR is available again - using the jar for now.
+        conda_envs:
+            mixcr: "{MODSDIR}/envs/mixcr"
+            
+        threads:
+            mixcr_run: 4
+
+        mem_mb:
+            mixcr_run: 2000
```

Comment:
Does it really not use more than this amount of RAM ever?

Reply:
I'm running the test data again with the `time -v` command Bruno showed yesterday and will update the value once the run is done.